components/form/form-helper.js

```diff
@@ -31,7 +31,8 @@ const _findFormElementsHelper = (ele, eles, isFormElementPredicate, visitChildre
 		eles.push(ele);
 	}
 	if (visitChildrenPredicate(ele)) {
-		for (const child of ele.children) {
+		const children = ele.tagName === 'SLOT' ? ele.assignedNodes() : ele.children;
+		for (const child of children) {
```

Comment:
For better or worse this is now going to find form elements inside slots that weren't previously being found...